deploy/examples/create-external-cluster-resources.py

```diff
@@ -652,6 +644,20 @@ def __init__(self, arg_list=None):
             self.cluster = rados.Rados()
             self.cluster.conf_read_file()
         self.cluster.connect()
+        if not self._arg_parser.rgw_pool_prefix and not self._arg_parser.upgrade:
+            self._arg_parser.rgw_pool_prefix = self.DEFAULT_RGW_POOL_PREFIX
+        # when user has not provided rgw pool name during upgrade, get the rgw pool name used during connection
```

Comment:
# get the rgw pool name from the client.healthchecker caps

Reply:
update